src/CommunityToolkit.Maui.SourceGenerators/Helpers/SourceGeneratorHelpers.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Text;
+
+namespace CommunityToolkit.Maui.SourceGenerators.Helpers
+{
+	static class SourceGeneratorHelpers
+	{
+		public static void FormatText(ref string classSource, CSharpParseOptions? options)
+		{
+			var mysource = CSharpSyntaxTree.ParseText(SourceText.From(classSource, Encoding.UTF8), options);
```

Comment:
Let's change this to something more descriptive like `sourceCodeText`.

We should void using `my` for variable names in production code; variable names like `myObject` are better purposed for samples/demos.